page-objects/petDetailsPage.ts

```diff
@@ -1,9 +1,42 @@
 import { expect, Page, test } from "@playwright/test";
 
-export class PetTypesPage {
+export class PetDetailsPage {
     readonly page: Page
 
     constructor(page: Page) {
         this.page = page
     }
+
+    async updatePetTypeTo(newPetType: string) {
+        this.clearOldAndFillNewPetTypeToInputField(newPetType)
+        await this.page.getByRole("button", { name: "Update" }).click();
+    }
+
+    async cancelUpdateOfPetType(newPetType: string) {
+        this.clearOldAndFillNewPetTypeToInputField(newPetType)
```

Comment:
This is a confusing approach.
This method should be responsible only canceling of pet type update. It should not include the clearing of the input field.
What if I need just to cancel the the pet type update without clearing the input field?
Also this approach makes test body less readable and less understandable about steps of test case.

Test case would look better if it would have:
```
await pm.onPetTypesPage().selectEditPetByIndex(1)
await pm.onPetDetailsPage().clearOldAndFillNewPetTypeToInputField('moose')
await pm.onPetDetailsPage().cancelUpdateOfPetType()
await pm.onPetTypesPage().validateRowValueByIndex(1, 'dog')
```